src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/util/ManagedIndexUtils.kt

```diff
@@ -486,3 +486,24 @@ fun Policy.isSafeToChange(stateName: String?, newPolicy: Policy, changePolicy: C
 
     return true
 }
+
+/**
+ * Disallowed actions are ones that are not specified in the [ManagedIndexSettings.ALLOW_LIST] setting.
+ */
+fun Policy.getDisallowedActions(allowList: List<String>): List<String> {
+    val allowListSet = allowList.toSet()
+    val disallowedActions = mutableListOf<String>()
```

Comment:
NP: To be a little more space efficient, you can declare this as a ```mutableSetOf<String>``` instead so that duplicate actions are not added into ```disallowedActions``` and then convert it to a list at the end by doing ```disallowedActions.toList()```

Reply:
Don't expect list to be large, will skip for now.